dart/lib/src/sentry_transaction_context.dart

```diff
@@ -21,12 +21,14 @@ class SentryTransactionContext extends SentrySpanContext {
     SpanId? parentSpanId,
     this.transactionNameSource,
     this.samplingDecision,
+    String? origin,
   }) : super(
           operation: operation,
           description: description,
           traceId: traceId,
           spanId: spanId,
           parentSpanId: parentSpanId,
+          origin: origin,
```

Comment:
Missing `SentryTransactionContext.fromSentryTrace` that should probably default to something else if it's not part of the context (which it's not part of the header anyway).
Eg `distributed`, if we agree on that, we should add it to https://develop.sentry.dev/sdk/performance/trace-origin/

Reply:
The only place the `SentryTransactionContext.fromSentryTrace ` is called from is from a test. As it is not used anywhere in the SDK, I don't hink we need an origin param. Is this API that is supposed to be used by users? If yes, we also don't need the origin as param,  as it is not supposed to be set by users, but we should set the value to 'manual' in this case.

Reply:
> Is this API that is supposed to be used by users?

yes, used by this feature https://docs.sentry.io/platforms/dart/performance/connect-services/
we do it automatically, but in case people wanna do it themselves, it's also ok.

> but we should set the value to 'manual' in this case.

that's what I wanted, just wondering if `manual` is the one or a new one would be better.

Reply:
If `fromSentryTrace` is only called from tests, I think we don't need to bother. To be clear: origin is not expected to be set manually by users.

Reply:
It's meant for the users, so technically would be nice to measure the usage of it.
We can default to `manual` for now since it already exists.

Reply:
If it stems from users I would use `manual` as the default.